ksqldb-functional-tests/src/test/resources/rest-query-validation-tests/insert-values.json

```diff
@@ -357,6 +357,18 @@
         {"topic": "test_topic", "key": 1, "value": {"V0": 0.00}}
       ]
     },
+    {
+      "name": "should insert decimals between zero and one",
```

Comment:
nit: rest query translation tests take much longer to run than the standard QTT tests. instead it may make sense to just add a QTT test where we use a decimal literal that's `1 > x > -1`

Reply:
Thank you! Will do that. I'm still figuring out how to run the feature tests selectively locally (this test is breaking anyway right now) -- will fix and update!

Reply:
Here are my configurations (the first runs all QTT tests, the second runs just a specific file). Also FYI there exists a `decimal.json` QTT test.

![image](https://user-images.githubusercontent.com/3172405/152578838-8582e53d-2cba-4d6b-8cde-9b1d8e2d54aa.png)
![image](https://user-images.githubusercontent.com/3172405/152578872-1590b2cf-94b3-41b2-9938-00864f07d30b.png)

I think similar things work for RQTT

Reply:
@agavra Would it be ok for me to steal these images and toss them in a debugging.md file?:)

Reply:
@agavra Thank you, this is super helpful! I arrived at the same IntelliJ dialog after a bunch of messing around, but your setup is a lot cleaner!

Reply:
@jnh5y go for it :)

Reply:
Interestingly the bug doesn't trigger on the query translation tests, only on the rest query translation tests. Leaving the rest-based test in here while I figure out exactly why the query translation tests don't trigger it...